physics/module_MYNNPBL_wrapper.F90

```diff
@@ -287,15 +288,17 @@ SUBROUTINE mynnedmf_wrapper_run(        &
 !smoke/chem arrays
    !   real(kind=kind_phys), dimension(:,:), intent(inout), optional ::   &                                     
    !  &         qgrs_smoke_conc, qgrs_dust_conc
-   !   real(kind=kind_phys), allocatable, dimension(:,:,:) :: chem3d
-   !   real(kind=kind_phys), dimension(:,:), intent(in), optional :: vdep
-   !   real(kind=kind_phys), dimension(:),   intent(in), optional :: frp, emis_ant_no
+      real(kind_phys), dimension(:), intent(inout) :: frp
+      logical, intent(in) :: mix_chem, fire_turb
+      real(kind=kind_phys), dimension(:,:,:), intent(inout) :: chem3d
+      real(kind=kind_phys), dimension(im)   :: emis_ant_no
+      real(kind=kind_phys), dimension(im,ndvel) :: vdep
 !for testing only
-      real(kind=kind_phys), dimension(im,levs)                      ::   &
-     &         qgrs_smoke_conc, qgrs_dust_conc
-      real(kind=kind_phys), allocatable, dimension(:,:,:) :: chem3d
-      real(kind=kind_phys), dimension(im,ndvel)      :: vdep    !not passed in yet???
-      real(kind=kind_phys), dimension(im)            :: frp, emis_ant_no
+!      real(kind=kind_phys), dimension(im,levs)                      ::   &
+!     &         qgrs_smoke_conc, qgrs_dust_conc
+!      real(kind=kind_phys), allocatable, dimension(:,:,:) :: chem3d
+!      real(kind=kind_phys), dimension(im,ndvel)      :: vdep    !not passed in yet???
+!      real(kind=kind_phys), dimension(im)            :: frp, emis_ant_no
 
```

Comment:
You can delete these lines if you want. They were only meant to be temporary, for testing prior to this commit.

Reply:
Joe, these have been deleted in the updated PR.

Reply:
thanks